reporter/src/main/kotlin/model/EvaluatedModelMapper.kt

```diff
@@ -334,7 +334,7 @@ internal class EvaluatedModelMapper(private val input: ReporterInput) {
 
     private fun addRuleViolation(ruleViolation: RuleViolation) {
         val resolutions = addResolutions(ruleViolation)
-        val pkg = packages.getValue(ruleViolation.pkg)
+        val pkg = packages[ruleViolation.pkg] ?: createEmptyPackage(ruleViolation.pkg)
```

Comment:
Commit message: `demanded the ID to by a valid project`
--> `demanded the ID to be a valid project`